index.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Quiz - o - Mania</title>
+    <link rel="stylesheet" href="https://argon-css.netlify.app/components/index.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.2/css/all.min.css"
+        integrity="sha512-HK5fgLBL+xu6dm/Ii3z4xhlSUyZgTT9tuc/hSrtw6uzJOvgRr2a9jyxxT1ely+B+xFAmJKVSTbpM/CuL7qxO8w=="
+        crossorigin="anonymous" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Dosis:wght@200;300;400;500;600;700;800&display=swap"
+        rel="stylesheet" />
+
+    <link rel="stylesheet" href="/index.css">
+</head>
+
+<body>
+    <!-- header -->
+    <header class="w-full h-20 shadow bg-grey-800 fixed z-50">
+        <div class="flex justify-around align-items-center h-full w-full">
+            <!-- logo -->
+            <div class="flex justify-center align-items-center">
+                <a href="/index.html">
+                    <h5
+                        class="h5 cursor-pointer mr-10 text-fuchsia-500 logo text-sm italic inline-flex justify-center align-items-center">
+                        Quiz - o - mania
+                    </h5>
+                </a>
+                <a href="/index.html">
+                    <h6 class="h6 mr-6 cursor-pointer text-white text-hover-fuchsia-500"> Home </h6>
+                </a>
+                <a href="/pages/quiz-page.html">
+                    <h6 class="h6 cursor-pointer text-white text-hover-fuchsia-500"> Quizzes </h6>
+                </a>
+            </div>
+            <!-- logo -->
+
+            <!-- searchbar -->
+            <form class="search__bar__container ">
+                <input type="text" class="search__bar" placeholder="Search">
+                <a href="#"><i class="fas fa-search search__icon text-fuchsia-500"></i></a>
+            </form>
+            <!-- searchbar -->
+
+
+            <!-- icons (profile) -->
+            <div class="flex justify-center align-items-center">
+
+                <a href="">
+                    <button style="background: transparent;"
+                        class="btn flex flex-col text-fuchsia-500 text-hover-fuchsia-400 tooltip">
+                        <span><i class="text-lg fas fa-user "></i> </span>
+
+                        <span style="text-transform: none;" class="tooltip-text">Profile</span>
```

Comment:
here too.

Reply:
its because i want it to overwrite one of the property my classes are applying, inline styles are more specific